include/mbedtls/ssl.h

```diff
@@ -2106,12 +2106,27 @@ void mbedtls_ssl_conf_cert_req_ca_list( mbedtls_ssl_config *conf,
 
 #if defined(MBEDTLS_SSL_MAX_FRAGMENT_LENGTH)
 /**
- * \brief          Set the maximum fragment length to emit and/or negotiate
- *                 (Default: MBEDTLS_SSL_MAX_CONTENT_LEN, usually 2^14 bytes)
+ * \brief          Set the maximum fragment length to emit and/or negotiate.
```

Comment:
Apologies, @k-stachowiak, for not noticing this earlier, but 2.7 doesn't have separate `MBEDTLS_SSL_IN_CONTENT_LEN` and `MBEDTLS_SSL_OUT_CONTENT_LEN`, but only `MBEDTLS_SSL_MAX_CONTENT_LEN`. Could you revert that change?